lib/Monitor.php

```diff
@@ -188,7 +188,7 @@ private function cleanParams($params)
 
     private function cleanMetrics($metrics)
     {
-        return array_map(function ($key) {
+        return array_map(function ($key) use ($metrics) {
```

Comment:
@dongm2ez thanks for catching this! Would you be able to add a test for `cleanMetrics`?

Reply:
Sure! today or tomorrow.